### ISSUE TITLE
Image Thumbnail Slider- Alt Text and Tooltip are Missing

### ISSUE THREAD

**Issue by Kummu007** [hlxsites/aldevron#135]
Hi Team, 

I have noticed that for the image thumbnail slider Alt text and the tooltip hover are missing. Please look into the below and do the needful.

![image](https://github.com/hlxsites/aldevron/assets/117751754/0cb99d45-d0e4-4cc7-a9b6-7f00bc40a520)

![image](https://github.com/hlxsites/aldevron/assets/117751754/02fff0d3-cc97-4103-b041-7c1cd031bb43)

**Comment by RAJSEKHARMANUBALLA** [comment]
This issue is fixed. Please make sure to author Alt text in the respective word document.  Screenshot for reference.

<img width="957" alt="alt text screenshot" src="https://github.com/hlxsites/aldevron/assets/148474340/315fecd9-d180-4f13-b3ae-69162aad759d">

**Comment by teshukatepalli1** [comment]
Hi @Kummu007  Looks like this issue is fixed by @RAJSEKHARMANUBALLA . Please validate and close

**Comment by teshukatepalli1** [comment]
Image alt can be given from doc itself

**Comment by Kummu007** [comment]
This is working as expected.

![image](https://github.com/hlxsites/aldevron/assets/117751754/2db8ab4d-00c1-4785-86f3-f9c906f1e391)
